digits/model/views.py

```diff
@@ -7,8 +7,10 @@
 import math
 import tarfile
 import zipfile
+import os
 
 import flask
+from flask import redirect, url_for
```

Comment:
Can you use the above import like `flask.redirect` and `flask.url_for`? That lines up with the coding style elsewhere in DIGITS.